recipes/usockets/all/conandata.yml

```diff
@@ -1,34 +1,42 @@
 sources:
-  "0.4.0":
-    url: "https://github.com/uNetworking/uSockets/archive/v0.4.0.tar.gz"
-    sha256: "f9f15b395def578cc79a5b32abc64fa9cff5dac873062911f515b984b90f7cc2"
-  "0.6.0":
-    url: "https://github.com/uNetworking/uSockets/archive/v0.6.0.tar.gz"
-    sha256: "999387d3653b2bc663c34aa7e973358ac4c4897dccd644553a5caab843a978a1"
-  "0.7.1":
-    url: "https://github.com/uNetworking/uSockets/archive/v0.7.1.tar.gz"
-    sha256: "1fdc5376e5ef9acf4fb673fcd5fd191da9b8d59a319e9ec7922872070a3dd21c"
+  "0.8.2":
+    url: "https://github.com/uNetworking/uSockets/archive/v0.8.2.tar.gz"
+    sha256: "c17fc99773a30552cdd7741ff98af177eeecb26b89fc38011b430956b3b2eca5"
   "0.8.1":
     url: "https://github.com/uNetworking/uSockets/archive/v0.8.1.tar.gz"
     sha256: "3b33b5924a92577854e2326b3e2d393849ec00beb865a1271bf24c0f210cc1d6"
-patches:
+  "0.7.1":
+    url: "https://github.com/uNetworking/uSockets/archive/v0.7.1.tar.gz"
+    sha256: "1fdc5376e5ef9acf4fb673fcd5fd191da9b8d59a319e9ec7922872070a3dd21c"
+  "0.6.0":
+    url: "https://github.com/uNetworking/uSockets/archive/v0.6.0.tar.gz"
+    sha256: "999387d3653b2bc663c34aa7e973358ac4c4897dccd644553a5caab843a978a1"
   "0.4.0":
-    - patch_file: "patches/0001-makefile.patch"
+    url: "https://github.com/uNetworking/uSockets/archive/v0.4.0.tar.gz"
+    sha256: "f9f15b395def578cc79a5b32abc64fa9cff5dac873062911f515b984b90f7cc2"
+patches:
+  "0.8.2":
+    - patch_file: "patches/0001-makefile_0.8.2.patch"
```

Comment:
since this is new please add patch fields ❤️

Reply:
Added patch fields for all versions